### ISSUE TITLE
Jenkins is configured to  start builds only after a minute

### ISSUE THREAD

**Issue by sthaha** [fabric8io/fabric8-build-team#46]
I see that jenkins is configured to wait for a minute before it starts a build? See [quiet period][quiet]. There is a good reason to do this but do we want to preserve this behaviour or should the builds start as soon as it is requested? 

[quiet]: https://jenkins.io/blog/2010/08/11/quiet-period-feature/

**Comment by chmouel** [comment]
👍 I think it may be good to have it on the openshift.io backlog and listed on https://github.com/openshiftio/openshift.io/issues/2314

**Comment by sthaha** [comment]
Closing this in favour of  https://github.com/openshiftio/openshift.io/issues/2495
